cmd/down.go

```diff
@@ -94,6 +95,10 @@ func runDown(ctx context.Context, dev *model.Dev) error {
 		return err
 	}
 
+	if err := diverts.Delete(ctx, dev, client); err != nil {
```

Comment:
can we rename to diverts.DeleteIfNeeded (or something like that) so it's more obvious that the method will be a no-op if there's no divert? the other option is to have the caller do that check. it's more verbose, but I think it helps with code readability.